api/genqlient.graphql

```diff
@@ -85,15 +85,16 @@ query GetBuild($uuid: ID!) {
   }
 }
 
-query GetScheduledBuilds($slug: ID!) {
-  pipeline(slug: $slug) {
+query GetScheduledBuilds($slug: ID!, $agentQueryRules: [String!]) {
+  organization(slug: $slug) {
     # @genqlient(pointer: true)
     id
     jobs(
       state: [SCHEDULED]
       type: [COMMAND]
       first: 100
       order: RECENTLY_ASSIGNED
```

Comment:
Jobs won't have an "agent assigned at" timestamp which is the "recently assigned" ordering while they are "scheduled", so I reckon this might be better as the default of "RECENTLY_CREATED". Ideally this should use the same ordering as agent dispatch, which is based on the job's "scheduled at", but I don't think we have a graphql cursor order option for that currently.

Reply:
Oh.. but RECENTLY_CREATED will be the opposite of what we want, I think. Frustrating, I don't think there is a better option right now. So maybe ignore the above for the moment.

Reply:
Yeah, I tested both orderings and `RECENTLY_ASSIGNED` seemed to be the one that gave me them in order created, confusingly enough